src/Cms/App.php

```diff
@@ -1615,7 +1615,6 @@ public function smartypants(string $text = null): string
 	 * @param mixed $name
 	 * @param array|object $data Variables or an object that becomes `$item`
 	 * @param bool $return On `false`, directly echo the snippet
-	 * @return string|null
 	 * @psalm-return ($return is true ? string : null)
 	 */
 	public function snippet($name, $data = [], bool $return = true, bool $slots = false): Snippet|string|null
```

Comment:
Any other value for `$name` will break in the component anyway, so it can already be blocked here:

```suggestion
	public function snippet(string|array|null $name, $data = [], bool $return = true, bool $slots = false): Snippet|string|null
```